### PR TITLE
Added typerandom/validator package to validators.

### PR DESCRIPTION
**Pull request by typerandom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -714,6 +714,8 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 
 *Libraries for validation.*
 
+* [validator](https://github.com/typerandom/validator) - A powerful validation library for Go.
+
```
Comment (mholt) [inline README.md]: Hate to be picky, but let's get rid of this empty line. :) Thanks

**Comment by mholt** [comment]
Could use more godoc

**Comment by matrixik** [comment]
Hi, @typerandom do you plan to add more documentation to typerandom/validator package? And one file is not formatted.
http://goreportcard.com/report/typerandom/validator

**Comment by cassiobotaro** [comment]
@typerandom  For  quality standards please follow:  https://github.com/avelino/awesome-go/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by cassiobotaro** [comment]
Please resend following the new guideline: https://github.com/avelino/awesome-go/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Pull request closed without merging**